reports/Texas.md

```diff
@@ -302,3 +302,32 @@ id: tx-sanantonio-2
 **Links**
 
 * https://twitter.com/SanAntonioProbz/status/1268027491987468288
+
+### Police gang up on protestors to beat them with batons. Then pepper spray man for recording | (believed to be) June 2nd
+
+Police isolate protester and then 4 to 5 officers beat him with a baton.  When one of them realizes he is being recorded, he pepper sprays the person recording to make him stop.
+
+tags: baton, pepper-spray, punch
+
+id: tx-houston-3
+
+**Links**
+
+* https://twitter.com/greg_doucette/status/1268392253665546245
+
+### Police fire teargas canisters and projectiles into peaceful crowd of unarmed protestors | (believed to be) May 31st
```

Comment:
Can you put this under El Paso? If it doesn't exist can you make a new El Paso alphabetically ordered

Reply:
ok, I see now. yep will re-order these

Reply:
@emarcey done